Tools/LyTestTools/tests/unit/test_artifact_manager.py

```diff
@@ -24,60 +24,66 @@
 ROOT_LOG_FOLDER = os.path.join("TestResults", DATE, "pytest_results")
 
 
-@mock.patch('os.makedirs', mock.MagicMock())
```

Comment:
I'm okay with this change, however the benefit of mocking at this class/suite level is that it assures that directory creation never gets accidentally added back in during test execution.